pandas/tests/plotting/test_hist_method.py

```diff
@@ -129,6 +129,21 @@ def test_plot_fails_when_ax_differs_from_figure(self):
         with pytest.raises(AssertionError):
             self.ts.hist(ax=ax1, figure=fig2)
 
+    @pytest.mark.parametrize(
+        "histtype, expected",
+        [
+            ("bar", True),
+            ("barstacked", True),
+            ("step", False),
+            ("stepfilled", True),
+        ],
+    )
+    def test_histtype_argument(self, histtype: str, expected: bool) -> None:
```

Comment:
pls remove annotations for tests. 

maybe i wasn't clear enough, I only meant to add annotation for the helper function you add in `common.py`.

Reply:
Done